Author: Jakub Wilk <jwilk@debian.org>
Description:
 Make file recognize byte-compiled files generated by Python 3.3
 (Closes: #697110).

diff -Naurp file.orig/magic/Magdir/python file/magic/Magdir/python
--- file.orig/magic/Magdir/python
+++ file/magic/Magdir/python
@@ -22,6 +22,7 @@
 0	belong		0x3b0c0d0a	python 3.0 byte-compiled
 0	belong		0x4f0c0d0a	python 3.1 byte-compiled
 0	belong		0x6c0c0d0a	python 3.2 byte-compiled
+0	belong		0x9e0c0d0a	python 3.3 byte-compiled
 
 0	search/1/w	#!\ /usr/bin/python	Python script text executable
 !:mime text/x-python
